src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,13 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+
+        return movieDao.get(id).orElseThrow(() -> new DataProcessingException(
+                "Can't get movie from DB by ID " + id, new RuntimeException()));
```

Comment:
we use DataProcessing on DAO level, on Service level let's use EntityNotFound
don't create new Runtime just for the sake of it, if there's nothing to rethrow, that's ok
Fix everywhere